boards/arm/lpcxpresso55s36/Kconfig.defconfig

```diff
@@ -0,0 +1,11 @@
+# LPCXpresso55S36 board
+
+# Copyright (c) 2022 NXP
```

Comment:
NXP guidance on Copyright is to drop the `(c)`

Reply:
Thanks for reminder.